journalbeat/plan.sh

```diff
@@ -0,0 +1,52 @@
+pkg_svc_user="root"
+pkg_name="journalbeat"
+pkg_description="Journalbeat is a log shipper from systemd/journald to Logstash/Elasticsearch"
+pkg_origin="core"
+pkg_version="5.5.2"
+pkg_maintainer="smartB Engineering <dev@smartb.eu>"
```

Comment:
I think the current pattern here is to say `"The Habitat Maintainers <humans@habitat.sh>"` and add yourself to the codeowners file for this plan.

Reply:
Can you point me at an example of a 'code owners' file?

Reply:
It's in the root of the project ;)

Reply:
Thanks :) - a lot has changes since the PR :)